### ISSUE TITLE
unclear doc example and template for OpenBMC when trying to define a new node 

### ISSUE THREAD

**Issue by NickyDaB** [xcat2/xcat-core#6417]
# Summary:
I'm not exactly sure. But I think @whowutwut thinks the example is bad, and it should be improved. 

# Subject in question
On this page: https://xcat-docs.readthedocs.io/en/stable/guides/admin-guides/manage_clusters/ppc64le/discovery/manually_define.html

# Context discussion

Originally posted in slack...

Comments from @whowutwut :

> In the OpenBMC case, it’s the last bullet, but that sample is ridiculous…

```
 mkdef -t node -o cn2 --template ppc64le-openbmc-template mac=66:55:44:33:22:11 ip=172.12.139.2 bmc=172.11.139.2 bmcpassword=USERID bmcusername=PASSW0RD
```

@whowutwut then asked @gurevich 
> Can we google search if `root/0penBmc` is out there, if it is publically defined, let’s default the template  to  it…
the more we can do for the user, the better off…
USERID/PASW0RD is default for x86_64 … IMMs… (edited) 
I’m also not sure why we FORCE the user to provide `bmcusername` and password, because it should take it from the  site  table…
that means we are overriding the site table values…  not good practice :( 

@gurevichmark replied with:

> 0penBmc pw is publicly defined:

![image](https://user-images.githubusercontent.com/4662139/64283973-83e46100-cf26-11e9-98ba-4ce823b60729.png)

Then @whowutwut  said: 

> @NickyDaB  pasted me this:

```
[root@c650mnp05 ~]# lsdef -t node --template ppc64le-openbmc-template
Object name: ppc64le-openbmc-template
    arch=ppc64le
    bmc=MANDATORY:The hostname or ip address of the BMC adapater
    bmcpassword=MANDATORY:the password of the BMC
    bmcusername=MANDATORY:the username of the BMC
    cons=openbmc
    groups=all
    ip=OPTIONAL:the ip address of the node
    mac=OPTIONAL:the mac of the node
    mgt=openbmc
    netboot=petitboot
    nodetype=mp
    postbootscripts=otherpkgs
    postscripts=syslog,remoteshell,syncfiles
    serialport=0
    serialspeed=115200
    usercomment=the template for openbmc controlled PowerLE NV node definition
[root@c650mnp05 ~]#
```

> so it looks like it’s `MANDATORY` …
I think `ip=OPTIONAL` is a mistake… and `mac=OPTIONAL` is a mistake
xCAT can’t provision the node without IP and MAC
So at least at a start if we require something the `openbmc-template` should deafult to  `root/0penBmc`
otherwise we are not helping the user out in any way

**Comment by NickyDaB** [comment]
also side note. I can't add labels to this issue.

**Comment by whowutwut** [comment]
@NickyDaB, can you modify the title to be specific to "OpenBMC" , otherwise the scope of this issue is way too large.

**Comment by NickyDaB** [comment]
@whowutwut more descriptive?

**Comment by NickyDaB** [comment]
also, lol, 

what's up with this example?

```
creating a node definition “cn2” with the template “ppc64le-openbmc-template”(openbmc controlled ppc64le node) shipped by xCAT

# mkdef -t node -o cn2 --template ppc64le-openbmc-template mac=66:55:44:33:22:11 ip=172.12.139.2 bmc=172.11.139.2 bmcpassword=USERID bmcusername=PASSW0RD
1 object definitions have been created or modified.

the unspecified attributes of newly created node “cn2” will be assigned with the default values in the template
```

specifically: 
```
bmcpassword=USERID bmcusername=PASSW0RD
```

seems backwards? or is that on purpose?

**Comment by whowutwut** [comment]
Yes, that is wrong, feel free to submit a PR and at least change  it to  the `root/0penBmc` values 👏

**Comment by gurevichmark** [comment]
Docs and templates fixed by #6450
